core/tests/protractor_utils/player.js

```diff
@@ -138,13 +138,37 @@ var openFeedbackPopup = function() {
   element(by.css('.protractor-test-exploration-feedback-popup-link')).click();
 };
 
+var openSuggestionPopup = function() {
+  element(by.css('.protractor-test-exploration-suggestion-popup-link')).click();
+};
+
 var submitFeedback = function(feedback) {
   openFeedbackPopup();
   element(by.css('.protractor-test-exploration-feedback-textarea')).
     sendKeys(feedback);
   element(by.css('.protractor-test-exploration-feedback-submit-btn')).click();
 };
 
+var fillInSuggestion = function(suggestion) {
+  var suggestionModalClass = '.protractor-test-exploration-suggestion-modal';
+  var richTextEditor = forms.RichTextEditor(
+    element(by.css(suggestionModalClass)));
+  richTextEditor.setPlainText(suggestion);
+};
+
+var addsuggestionDescription = function(description) {
```

Comment:
Argh, whoops -- I forgot this hadn't been addressed before merging. I'll submit a PR for it (and be more careful next time about giving preliminary LGTMs)...